index.js

```diff
@@ -51,11 +51,9 @@ exports.aceAttribClasses = function(hook_name, attr, cb){
 */
 
 // Add the props to be supported in export
-exports.exportHtmlAdditionalTags = function(hook, pad, cb){
-  cb(sizes);
-};
+exports.exportHtmlAdditionalTags = async (hookName, pad) => sizes;
 
-exports.getLineHTMLForExport  = function (hook, context, cb) {
+exports.getLineHTMLForExport = async (hookName, context) => {
```

Comment:
Oh boy, is this gonna need to be done on every plugin that uses getLineHTMLForExport? 

If so, I got some work to do...

Reply:
We already have an async method no?

https://github.com/ether/ep_font_size/pull/17/files

``asyncLineHTMLForExport``

Reply:
> Oh boy, is this gonna need to be done on every plugin that uses getLineHTMLForExport?
> 
> If so, I got some work to do... It's also worth noting this will break the plugin on older versions of Etherpad no? Which is a huge problem in itself....?? But I guess it's broken since ~1.8.4 anyway...

Instead of `async` the plugins can do `return cb(foo);` (not `cb(foo); return;`) and they'll be compatible with new and old Etherpad (I think; I'm not very familiar with how it was before). This works on newer Etherpad because `cb(foo)`returns a Promise that resolves to `foo`, and it works on old Etherpad because the return value is ignored (I think).

> We already have an async method no?
> 
> https://github.com/ether/ep_font_size/pull/17/files
> 
> `asyncLineHTMLForExport`

Huh, I didn't notice that. Does it still exist in 1.8.6? It so, the `aCallAll` for `getLineHTMLForExport` should probably be changed to `callAll`.

Reply:
> Instead of `async` the plugins can do `return cb(foo);` (not `cb(foo); return;`) and they'll be compatible with new and old Etherpad (I think; I'm not very familiar with how it was before). 

Of we can modify `aCallAll` to work with `cb(foo); return;`.

Reply:
```
jose@ryzin:~/etherpad-lite/src/node$ grep -rni asyncLineHTMLForExport .
jose@ryzin:~/etherpad-lite/src/node$ 
```

I don't see it so maybe not.  Will modify plugins to ``return cb(foo)...``